ext/settings.html

```diff
@@ -3394,6 +3394,36 @@ <h1>Yomitan Settings</h1>
                     <td><code class="anki-field-marker">{document-title}</code></td>
                     <td>Title of the web page that the term or kanji appeared in.</td>
                 </tr>
+                <tr>
+                    <td><code class="anki-field-marker">{frequencies}</code></td>
+                    <td>
+                        Frequency information for the term or kanji.
+                    </td>
+                </tr>
+                <tr>
+                    <td><code class="anki-field-marker">{frequency-harmonic-rank}</code></td>
+                    <td>
+                        The harmonic mean of frequency data for the current term or kanji. Defaults to 9999999 when no frequency is found.
```

Comment:
This seems identical to the description right below it, but it should probably explain something about what the numbers mean since they are seemingly semantically different

Reply:
Added some rough labels to what 0 and 9999999 mean but I'm not exactly sure what the best way to explain these would be if we want a real explanation. Will think about it.